src/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -77,6 +76,12 @@ public void draw(final Rectangle bounds, final GameData data, final Graphics2D g
     final Image img = m_uiContext.getUnitImageFactory().getImage(type, owner, data,
         m_damaged > 0 || m_bombingUnitDamage > 0, m_disabled);
     graphics.drawImage(img, m_placementPoint.x - bounds.x, m_placementPoint.y - bounds.y, null);
+    if(!type.getName().equals("aaGun") && !type.getName().equals("factory")){//TODO add every "static" non-movable unit type of every map.
```

Comment:
I wonder if we really want to exclude AA Gun - they can move during non-combat. It can be pretty easy to have a couple that are next to each other and make it difficult to tell apart.